### PR TITLE
[infra-monitoring] Added vasa-exporter to infra-monitoring

### PR DESCRIPTION
**Pull request by christopherhans**
Helm charts for the vasa-exporter.

### CODE REVIEW

**Comment by auhlig** [comment]
Could you add the exporter [here](https://github.com/sapcc/helm-charts/tree/master/prometheus-exporters) and make sure the tests are green @christopherhans?

**Comment by christopherhans** [comment]
will do

**Comment by christopherhans** [comment]
@auhlig moved vasa-exporter in the correct directory and all tests are green.

**Comment by christopherhans** [comment]
@viennaa please have a look.

File: prometheus-exporters/vasa-exporter/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: vasa-exporter - Prometheus exporter for the VASA provider
+name: vasa-exporter
+version: 1.0.0
```
Comment (viennaa) [inline prometheus-exporters/vasa-exporter/Chart.yaml]: please add maintainer

File: prometheus-exporters/vasa-exporter/values.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+vasa_exporter:
+  enabled: true
```
Comment (viennaa) [inline prometheus-exporters/vasa-exporter/values.yaml]: default case is false

File: prometheus-exporters/vasa-exporter/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{{- if .Values.vasa_exporter.enabled }}
+apiVersion: v1
+kind: Deployment
+metadata:
```
Comment (viennaa) [inline prometheus-exporters/vasa-exporter/templates/deployment.yaml]: should look like this:
```
metadata:
  name: {{ include "infraMonitoring.fullname" . }}
  labels:
    app: {{ include "infraMonitoring.name" . }}
    chart: {{ include "infraMonitoring.chart" . }}
    release: {{ .Release.Name }}
    heritage: {{ .Release.Service }}
    system: infra_monitoring
```

File: prometheus-exporters/vasa-exporter/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  name: vasa-exporter
+  namespace: infra-monitoring
+
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+         app: vasa-exporter
```
Comment (viennaa) [inline prometheus-exporters/vasa-exporter/templates/deployment.yaml]: please change to this:
```
       app: {{ include "infraMonitoring.name" . }}
        release: {{ .Release.Name }}
```

File: prometheus-exporters/vasa-exporter/templates/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    spec:
+      volumes:
+        - name: config
+          configMap:
+            name: vasa-exporter
+      containers:
+         - name: vasa-exporter
```
Comment (viennaa) [inline prometheus-exporters/vasa-exporter/templates/deployment.yaml]:
```suggestion
         - name: {{ include "infraMonitoring.name" . }}
```

File: prometheus-exporters/vasa-exporter/templates/service.yaml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+{{- if .Values.vasa_exporter.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: vasa-exporter
+  annotations:
```
Comment (viennaa) [inline prometheus-exporters/vasa-exporter/templates/service.yaml]: annotations are not needed

File: prometheus-exporters/vasa-exporter/templates/service.yaml [hunk 7]
```diff
@@ -0,0 +1,15 @@
+{{- if .Values.vasa_exporter.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
```
Comment (viennaa) [inline prometheus-exporters/vasa-exporter/templates/service.yaml]: please change to:
```
metadata:
  name: {{ include "infraMonitoring.fullname" . }}
  labels:
    app: {{ include "infraMonitoring.name" . }}
    chart: {{ include "infraMonitoring.chart" . }}
    release: {{ .Release.Name }}
    heritage: {{ .Release.Service }}
    system: infra_monitoring
```

**Comment by viennaa** [comment]
please move it all:
https://github.com/sapcc/helm-charts/tree/master/system/infra-monitoring/vendor

**Pull request merged**